custom/opm/filters.py

```diff
@@ -51,6 +70,56 @@ def get_hierarchy():
     return hierarchy
 
 
+def user_data_as_hierarchy():
```

Comment:
Do you think this could be a source of inconsistencies between the three opm case reports?